src/App.jsx

```diff
@@ -14,33 +17,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
+  const [showReversed, setShowReversed] = useState(false);
```

Comment:
With  'show'  in the name looks like it's a function, but it's not. would be better 'isReversed'

Reply:
> With 'show' in the name looks like it's a function, but it's not. would be better 'isReversed'

Good point, boolean variables should start from `is` or `has`